sdk/servicebus/azure-servicebus/tests/async_tests/test_queues_async.py

```diff
@@ -231,7 +231,7 @@ async def test_async_queue_by_queue_client_conn_str_receive_handler_receiveandde
 
             assert not receiver._running
             assert len(messages) == 10
-            time.sleep(30)
+            time.sleep(10)
```

Comment:
can we ensure this will not increase the number of transient failures? same question for all similar changes

Reply:
🤔 hmm, that's a good point. I'm not sure that this won't increase transient failures. Do you know what the best way to check that would be?

Reply:
@rakshith91 thanks for being thoughtful!

as far as I could remember, the transient failures are usually caused by tests including receiving which doesn't wait long enough.

I think we could merge the PR first and see how it goes, if we keep observing failures, then we fix them :P

Reply:
sounds good to me!